### DISCUSSION TITLE
Interrupt cell execution

### DISCUSSION THREAD

**Discussion by kbkartik** [fonsp/Pluto.jl#967] [Questions about using Pluto]
I am new to Julia and Pluto.

I am using Julia (v1.5.3) with Pluto (v0.12.21) and PlutoUI (v0.7.2). I wrote some faulty code due to which cell keeps running. How can I interrupt the cell execution? Trying `Ctrl+Q` doesn't help.

**Comment by fonsp** [comment]
Can you split your two questions into two messages?

**Reply by kbkartik** [reply]
Updated

**Comment by fonsp** [comment]
Unfortunately we can't interrupt on Windows, see #452

**Reply by kbkartik** [reply]
Actually, I am running Julia and Pluto on Ubuntu using Windows Subsystem for Linux (WSL2). Does this issue still apply?

**Reply by fonsp** [reply]
It does not. If you find a **reproducible** example of a cell that does not interrupt, please post it to https://github.com/fonsp/Pluto.jl/issues/228
